pgjdbc/src/main/java/org/postgresql/jdbc/TimestampUtils.java

```diff
@@ -692,14 +697,13 @@ public synchronized String toString(LocalDate localDate) {
   }
 
   public synchronized String toString(LocalTime localTime) {
-    if (LocalTime.MAX.equals(localTime)) {
-      return "infinity";
-    } else if (LocalTime.MIN.equals(localTime)) {
-      return "-infinity";
-    }
 
     sbuf.setLength(0);
 
+    if (localTime == LocalTime.MAX) {
```

Comment:
@davecramer 
Would it be better to do `LocalTime.MAX.equals(localTime)` here, instead of using the reference equality?

Otherwise there will be a difference in write behavior between a LocalTime that has a value of `23:59:59.999999999` and when using `LocalTime.MAX`, even though they are value-equal.

Reply:
good point, thx